### DISCUSSION TITLE
Modeling the recent events before the game start (warning: spoilers)

### DISCUSSION THREAD

**Discussion by NetSysFire** [CleverRaven/Cataclysm-DDA#60406] [General]
Hey @I-am-Erk and all the other folks involved with lore, I think there is one major area that needs some love: Stuff that happened after the blob reached global groundwater saturation. It currently feels so unreal and abstract, as if something is missing.
To get straight to the point: I think there is some crucial missing data and this is not something I can figure out by myself. I'd like info on:

- How many people are dead/zombified at any given time (perhaps as a graph if you are feeling fancy)
- How many people are feral, could be integrated into the above
- How the peoples daily lives are affected/changing. As a player you usually do not see the government-level intel stuff, but you are just a normal human and it would help tremendously if there would be some notes on what the surviving population sees.
  - E.g: January: everything is still mostly normal.
  - March: Some people complain the grocery store was closed after a "riot" happened there, foodplace now employs more security staff to keep their restaurants orderly.
  - April: Stuff begins to break apart, many people do not even come into work anymore, sometimes remaining survivors band together to create a futile "bastion", sometimes people actively go out of their way to get around the now mostly closed stores and raid
  - End of April: 99% of the population is dead. You are one of the survivors. 99% might not be the intended figure to go after?
- A more fine-grained scale on feralness, if possible
  - Mostly asking because I wonder if the world could be livened up a bit if there is zombie fodder, like a lone crazy fighting zombies with a stick (even mentioned in the design doc), which will eventually turn feral if not killed before.
  - Or maybe people who holed up in their basement, fortified it and now refuse to come out, without having a stockpile of supplies (irrational decision because blob psychosis), could maybe even trade
  - Also people who just run away and also turn feral in their monster evolution pretty quickly.
  - E.g the stick-warrior and the fleeing dude could be placed in the zombie monstergroup, so most of it is still zombies but there is the occasional crazy survivor who will die very soon.
 
I can not really give a good example for the daily lives and feral things because knowing how many people are dead/feral/zombies is basically a prerequisite for them, so I had to guess.

**Comment by bombasticSlacks** [comment]
Something I mentioned in passing is to showcase some of this in game we could possibly do (this is taken from discord):

"If we really wanted to "go for it" we should do a mind web similar to a game like pathologic 2. 

When you start you have a few thoughts:

"I need the tools to survive"
"I need to be able to defend myself"
"Other people must have survived, maybe they are organizing" 
"How did this happen? There must be answers somewhere"

Then as you progress more thoughts appear. So under Defend myself after a week you could get the thought:

"The zombies seem to be changing and getting more powerful, I'll need to find ways to adapt"

After meeting the refugee center under others:
"The refugees make good trade partners but their situation is fragile"

After finding the badge for the TCL under "what happened":
 "this facility seems out of place, I wonder what they were doing there."

This would be a lot of work but as faction narratives start to intermingle would be useful for tracking state and history. It also gives new players hints on what to do *and* has the benefit of highlighting something's significance. Since certain portions of the game are currently under development and under developed."
